packages/core/src/sync/index.ts

```diff
@@ -1497,27 +1541,19 @@ export const getLocalSyncProgress = async ({
     cached === undefined
       ? [
           rpc.request({ method: "eth_chainId" }),
-          _eth_getBlockByNumber(rpc, { blockTag: "latest" }),
```

Comment:
Just noting that this will probably change the experience slightly for folks who have a totally broken / misconfigured RPC, because the first request will now happen during build.